docker-compose.yml

```diff
@@ -24,6 +24,75 @@ services:
+        WXSvsAl+hfQzWGmPnuYNtaRYFdJ7F36Grhpy/7Z5JAJ7O5I20SdAzjrBB8ZMo/y9
+        Nq1FTq0KKucav1xivCsRM0F14ga6oq6Sl+M9t5J8l8j7e2vklTnJpniiJbtjAuby
+        53tSjxSJ4rhxquGn6/bmDl0AYIgJGSqmMcUe/sqFAl7xKTIxjWfuiHxV1xjPxH+Q
+        uyGBEgbA/Pgn0iEji9G+GjXHjXr7LTWQ+GcqKMi/AYgwLzLeBbf8gqF4jYjMVY2T
+        4rbAh4fnJBa+Z7R5c4RdyVuumkT5SBa+y2ESRi/QKvNYZhucD6StbrIzbCQoHuzb
+        r0omHsH3e7sftP0CggEBAMH6b/0vN7kTIznLM+GWgRw7agoKTADf/NKH1BFxw61a
+        IXa44rfaSbhdgGboK7mk6gQY0h7M2Ej1Ni7a4j368w2fli7XSXNY4Fs+2E3hSwqw
+        i54+yKVa5AoNFvm67zD5UN6dNhY16HtA5CdLA/EKdxMIgJb1uQQDWFRmI5ZOFp6b
+        z1qNJNaNoxi7I9ibR/LpFnlXbOordq3dH2tEmPT5NO8gN7/96/mwnD2018ZS1O9v
+        yjjbxLmXCwAxNmzsvdr7bomlr4mliiCEwO9d94ktiPINz7Q75ErA/6iOK+BYsj1A
+        xzOMchiwyJEfVVKOy/Fagrpp5fhc2wGHNOJEnBfCkLkCggEBAIKsQgjskveAN7rf
+        pIMMzvRvdjoIkw5pwmnBlrmlGPnvtyoD8bxUud8khfTePcx+H9IynqEywtK74cyp
+        TH9sST60a5p6dqO49+cZQ41b0XsN05Ks1S46y2qAa6ApWsNSzOjvifXG1bwYTvEM
+        u6ZVOY83bQhwJsGz0mrVH4PgICMQsFL6lhzXkfgYgYzLvv0tKZ5UqXgsiZ54eYP7
+        edNnFkrBfhNAI8Oxa+tZgzS3HffvGDMVXTTD3gcASe0maPgS/xQYAoKFQYDKlk8q
+        yT2wUfwFC5nsZal/jH5uoAqhwQ2m1P9AlDuDQ6bn5Ts++GDuOPPj/AqbkgRQw0AB
+        Y0ucoXkCggEBALfrt+Fssr31ilHS/KcpZ9OX9gv9BWFS/0my5f2AS5ODQPVrifPe
+        skVvyj5IZqUH26b0zc3MVkqezbgRa7Buz6OcK+7Py+HlZBy/8qGtOtSw3kC1sNMv
+        4cYx8RXqC774tZfNoinfBUZQGcOz2dHjJVHTnvEDnWBTt52Wlql98u6CN1HZQBN6
+        iij5z5wiZ07HeLqmjSte+JETdyXjDdBAtdMBTjrpDKRbs1jCrlBp22IHMctFfTqn
+        HxQYlU0kD0OhYIZ8pXqyjcwVc0XCZ+EWXqF7PPPcUbIbWoy7+JgoIR/42qdixA/r
+        FtCaz/COWorGUetkW1to13KUey0K9qGEpmkCggEAcvO4CPXtLPNex5OCTXzeIlac
+        QapxMhQCzQh1eXET5/Gk4nug2Y8tSg448LT/rt3xWzjPx33hHvkYoVK6fLiu7/wW
+        6WfDlnkKKZO7NqtJD0G8nhV5hZ/2Wh4+35fXqC0BE57roHbgJwjOluLejTjAjz9u
+        VPZThb4kjez+POGEi88lqreKzSXMM7XPcYBtVTeAPVPexawoLggkI7fima2poOTE
+        krRUKO810Y74seifkp9MdG3hjIHjs/SFQ0jMckRS9/uN7vbfWuewhQXtsOHygnOg
+        sdiyApf8QvSMDF+oTpedtHRIvlkMEyhs8wGnhvw1PQefBmGV1rJAFi7t4JPqdw==
+        -----END RSA PRIVATE KEY-----
+
+      - |
+        RSA_PUBLIC_KEY=-----BEGIN PUBLIC KEY-----
+        MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEApHTFRuG90V1nncnKrelx
+        9pEqQlV+Li7LjkPWX7LQylkXtaWDqut+5zResjOTdCuvP80XkpZvF+Dqi4w5UKT5
+        wCbeF4MlSXpfG4fs+vRP83cG4kVJEtH6SY5cTh6eCLPqpQT8/JxfAmk0JORlWhpf
+        /1t4XiWTzKyDSxip3G+RLhsz80L9WmiXTeZtaEfASLV1YarjFTAk4ZKbJ4SNyIBt
+        eu8QY8hOtPxcTCrBJJct4QPeFZL2zRiqfDSLl9iJCSNjxPYYhKt5qMHXqXUHjbML
+        gXYdRApDZzyy5eOd9pfyD3p0ZGokXVrVOrKtJc7QybGABI6mr1rWFGd28oJXj4w5
+        e/ju9LC9Wfd4Sy1Hb5m1j/NExvn9Di5A9nlFebPnN9K2qPcIkzP51uGB3xtMRxua
+        383gXsnXmTIYvrgfiK7BgxBjjlXDMUZKcPIw1ZVXnl+BdKgDlpTMjpGqcxwV+BQf
+        8ERK4NC46+EVapYiBbOoc0Bwovrp2bIZ6ljpJn+E3W74WW44I6WFbK2cFQ6xnzAl
+        NTK64KzsVz+7FYBS7RrAN3mkg3GU6duPFfwoYPNAtm2S6IyGc3Z7q3IljLCljZuw
+        6XL5JklWcLsb8FSYy+adff0Fu7fo7i1DGj9PeJZWzjQlWYd73mNcYus6SCH2kpSk
+        fuXEyWJoEoFZnbSXIfJyGtUCAwEAAQ==
+        -----END PUBLIC KEY-----
```

Comment:
Added like that, because docker doesn't load multilines properly form env files: https://github.com/docker/compose/issues/5325